Indexing-Contract-Data-TheGraph.md

```diff
@@ -772,8 +772,7 @@ export default function Home() {
             type="number"
             className={styles.input}
             onChange={(e) => {
-              // The user will enter the value in ether, we will need to convert
-              // it to WEI using parseEther
+              // The user will enter the value for max players
```

Comment:
Good find! However, since we're on this, can you make it a little bit more friendly, as in, something like :

```js
// The user enters the maximum players that can join a game
```

Not compulsory to do this.

Reply:
updated as suggested